src/renderer/components/wallet/settings/Settings.tsx

```diff
@@ -25,7 +25,8 @@ type Props = {
   lockWallet?: () => void
   removeKeystore?: () => void
   retrieveLedgerAddress: ({ chain, network }: LedgerAddressParams) => void
-  removeLedgerAddress: ({ chain, network }: LedgerAddressParams) => void
+  removeLedgerAddress: (chain: Chain) => void
+  resetAllLedgerAddress: () => void
```

Comment:
Just curious: What's the reason to `remove` and / or to `reset` a Ledger?

Reply:
@sarawut11 [As discussed at Slack](https://thorchain.slack.com/archives/C012CLYR1DG/p1606813903105200?thread_ts=1606748252.098300&cid=C012CLYR1DG): Rename `reset*` to `remove*` (to avoid missunderstandings).